packages-next/admin-ui/src/pages/ListPage/index.tsx

```diff
@@ -77,11 +77,23 @@ let listMetaGraphqlQuery: TypedDocumentNode<
 function useQueryParamsFromLocalStorage(listKey: string) {
   const router = useRouter();
   const localStorageKey = `keystone.list.${listKey}.list.page.info`;
+  const cacheableQueries = ['sortBy', 'fields'];
```

Comment:
Can we not call this "cacheable"? When I think of a cache, I think storing things so we don't have to compute them again, not storing user stuff to help them